src/sentry_plugins/asana/plugin.py

```diff
@@ -232,8 +232,8 @@ def get_config(self, *args, **kwargs):
         ]
 
     def view_autocomplete(self, request: Request, group, **kwargs):
-        field = request.GET.get("autocomplete_field")
-        query = request.GET.get("autocomplete_query")
+        field = request.GET["autocomplete_field"]
+        query = request.GET["autocomplete_query"]
```

Comment:
should this be wrapped in a try except in case the key doesn't exist? what is wrong with request.GET.get?

Reply:
`.get(...)` is nullable -- which would always crash further on (especially the `.decode(...)` call)

Reply:
i see, we'd rather it fail here and be obvious

Reply:
indeed -- I think a further improvement would be to use an api serializer of some sort and produce a "X field is missing" message -- but shrugs